stable/atlantis/templates/statefulset.yaml

```diff
@@ -84,7 +84,7 @@ spec:
           {{- end }}
           {{- if .Values.allowForkPRs }}
           - name: ATLANTIS_ALLOW_FORK_PRS
-            value: {{ .Values.allowForkPRS }}
+            value: "{{ .Values.allowForkPRs }}"
```

Comment:
I think `{{ | quote }}` is better practice

Reply:
Thanks @jeff-knurek. Updated